### ISSUE TITLE
Search other name tags

### ISSUE THREAD

**Issue by yotamN** [CanalTP/mimirsbrunn#276]
When I search for a street with a loc_name or alt_name I can't find it. I can only find it using the name tag. There are over 800,000 places with alt_name and over 90,000 places with loc_name. It would improve the search

**Comment by antoine-de** [comment]
you're right, it might be really interesting.

It's not a trivial change though, we'll have to tweak/bench the ElasticSearch weight of the `loc_name`/`alt_name` to see if this does not inder the results.

Do you have some examples of stuff you want to find ?

**Comment by yotamN** [comment]
Yes, this street for example have both a loc_name and a alt_name (in Hebrew):
https://www.openstreetmap.org/way/522392474

When I search for הרחבה or הרחבת בנים I expected to find it.

**Comment by TeXitoi** [comment]
we have also some needs on old_name https://www.openstreetmap.org/way/184551793

**Comment by yotamN** [comment]
There are a lot of name tags to consider (https://wiki.openstreetmap.org/wiki/Names).
Even `name_1` and `name:en1` that are deprecated have over 800,000 and over 20,000 usages respectively.
